x-pack/test/security_solution_endpoint/apps/endpoint/index.ts

```diff
@@ -4,13 +4,26 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import { FtrProviderContext } from '../../ftr_provider_context';
+import {
+  skipIfNoRegistry,
+  isRegistryEnabled,
+} from '../../../security_solution_endpoint_api_int/registryry';
+
+export default function (providerContext: FtrProviderContext) {
+  const { loadTestFile, getService } = providerContext;
 
-export default function ({ loadTestFile, getService }: FtrProviderContext) {
   describe('endpoint', function () {
     this.tags('ciGroup7');
     const ingestManager = getService('ingestManager');
+    const log = getService('log');
+
+    skipIfNoRegistry(log);
```

Comment:
We're still discussing whether or not we'll skip our tests. If we decide not too I'll remove these.